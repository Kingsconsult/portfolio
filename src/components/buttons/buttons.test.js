import React from 'react';
import renderer from "react-test-renderer";
import Button from "./index";
import snapshotDiff from "snapshot-diff";
import { ThemeProvider } from "styled-components";
const theme = {
  mode: "light"
};
// test("snapshot difference between 2 strings", () => {
//   expect(snapshotDiff(a, b)).toMatchSnapshot();
// });


// test("snapshot difference between 2 React components state", () => {
//   expect(
//     snapshotDiff(<Button test="say" />, <Button test="my name" />)
//   ).toMatchSnapshot();
// });

it("renders correctly when called ", () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={theme}>
            <Button />
      </ThemeProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});