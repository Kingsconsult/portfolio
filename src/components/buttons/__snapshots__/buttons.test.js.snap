// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when called  1`] = `
<button
  className="sc-AxirZ byZQPF"
  size="small"
/>
`;
